lib/core/test/integration/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -715,3 +715,34 @@ spec = do
             [ expectResponseCode HTTP.status403
             , expectErrorMessage errMsg403InvalidTransaction
             ]
+
+    it "TRANS_CREATE_09 - 0 amount transaction is forbidden on multi-output tx" $ \ctx -> do
+        wSrc <- fixtureWalletWith ctx [10_000_000, 12_000_000]
```

Comment:
What's the need for having the output defined as such? (asking because `fixtureWalletWith` needs to restore a 2 wallets & make two transactions, which is quite expensive (in terms of time) to do).

I don't think it's needed for the test here and, going for `fixtureWallet` should be enough, don't you think?

Reply:
OK, no particular reason for that. Will change to `fixtureWallet`.